amacc.c

```diff
@@ -1568,7 +1604,7 @@ int elf32(int poolsz, int *main)
      * (4 instruction * 4 bytes), so the first codegen and second codegen
      * have consistent code_size.
      */
-    int FUNC_NUM = EXIT - OPEN + 1;
+    int FUNC_NUM = MOD - OPEN + 1;
```

Comment:
Don't touch EXIT, please make EXIT the last system call so you won't need to add bellow `case DIV` and `case MOD`